tm2/pkg/crypto/keys/client/addpkg.go

```diff
@@ -66,9 +60,6 @@ func (c *addPkgCfg) RegisterFlags(fs *flag.FlagSet) {
 }
 
 func execAddPkg(cfg *addPkgCfg, args []string, io *commands.IO) error {
-	if cfg.pkgPath == "" {
-		return errors.New("pkgpath not specified")
-	}
```

Comment:
we can make the migration smoother by just adding a warning here with a comment to remove later.